### ISSUE TITLE
Not getting hashrate

### ISSUE THREAD

**Issue by Lepermessiahe2** [NebuTech/NBMiner#861]
In windows 10 I am not getting hashrate on LHR cards. Those LHR cards run lower hash rate than v40.1 w/74% unlock. I lose 20mhs on new version vs 40.1.

I have 6 nvidia cards and 1 amd card in the rig.

**Comment by Lepermessiahe2** [comment]
Overclocks do not affect hashrate on LHR cards except they will go lower but not higher.

**Comment by Lepermessiahe2** [comment]
Yes in v40.1 on 3070ti I will get 58mh on v41.3 I will get 44mh and cannot get it higher regardless of overclock settings. It will go down from 44mh but not anywhere above it. Sorry I was not specific.

**Comment by jh2663** [comment]
did you run the script as administrator? Otherwise, it won't unlock LHR.

**Comment by Lepermessiahe2** [comment]
That worked running as admin. Just had to add the file location to bat file to get miner to boot in admin mode, otherwise it would not boot in admin
